code/index.html

```diff
@@ -24,24 +24,45 @@ <h1>Does the person have</h1>
 
       <select id="questions">
         <optgroup label="hair color">
+          <option value=" ">select</option>
           <option value="brown">brown hair</option>
           <option value="yellow">yellow hair</option>
-          <!-- Add the rest of hair colors as options  -->
+          <option value="black">black hair</option>
+          <option value="purple">purple hair</option>
+          <option value="orange">orange hair</option>
+          <option value="grey">grey hair</option>
+          <option value="white">white hair</option>
+          <option value="hidden">hidden hair</option>
         </optgroup>
+
         <optgroup label="eye color">
           <option value="hidden">hidden eyes</option>
-          <!-- Add the rest of eye colors as options  -->
+          <option value="blue">blue eyes</option>
+          <option value="brown">brown eyes</option>
+          <option value="green">green eyes</option>
         </optgroup>
+
+        <optgroup label="character">
+          <option value="friendly">friendly</option>
+          <option value="aggressive">aggressive</option>
+          <option value="confused">confused</option>
+          <option value="thoughtful">thoughtful</option>
+        </optgroup>  
+
         <optgroup label="accessories">
           <option value="glasses">glasses</option>
-          <!-- Add the other accessory option here. (hat)  -->
+          <option value="hat">hat</option>
         </optgroup>
+        
         <optgroup label="other">
           <option value="smoker">a smoking habit</option>
         </optgroup>
       </select>
```

Comment:
Maybe it would be a nice think to generate long lists of options with your Javascript? It is nothing wrong, with this approach, but It makes you type a lot 😸